src/content/docs/accounts/accounts-billing/new-relic-one-user-management/authentication-domains-saml-sso-scim-more.mdx

```diff
@@ -168,12 +168,31 @@ At the bottom of the [authentication domain UI](#ui), you can control some other
 * Amount of idle time before a user's session expires. 
 * [User upgrade requests](#user-upgrade)
 
+
 ## User upgrade requests [#user-upgrade]
 
-At the bottom of the [authentication domain UI](#ui), one of the settings is controlling how basic users upgrade to billable [full users](/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure). You have two choices: 
+At the bottom of the [authentication domain UI](#ui), one of the settings is controlling how basic users upgrade to billable [full users](/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure). You have different options depending on whether you are managing user type in New Relic or via SCIM: 
```

Comment:
"user type" is used here and elsewhere. It seems like you should be managing "user types", but given that I'm seeing this in more than one place, I'm thinking it might deliberate?

Reply:
yeah it's referring to "user type" as a concept, not as individual things. Like saying "managing authentication method" instead of "managing authentication methods."